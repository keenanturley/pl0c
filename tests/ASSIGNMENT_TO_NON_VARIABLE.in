const p = 7;
var x, w;
begin
   x:= 4;
   w:= 7;
   if w > x then
	w:= w + 1
   else
	w:= x;
    p:= x;
end
