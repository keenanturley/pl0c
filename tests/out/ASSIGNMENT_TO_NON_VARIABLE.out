Lexeme List:
28 2 p 9 3 7 18 21 2 p 20 3 10 18 22 19 

Lexeme List (Symbolic Representation):
constsym identsym p eqsym numbersym 7 semicolonsym beginsym identsym p becomessym numbersym 10 semicolonsym endsym periodsym 

Error: Attempted to assign value to non-variable identifier.
